grails-app/conf/Config.groovy

```diff
@@ -257,18 +257,13 @@ portal {
 
         dataFile = [
             "WWW:DOWNLOAD-1.0-http--download",
-            "WWW:DOWNLOAD-1.0-http--downloaddata",
-            "WWW:DOWNLOAD-1.0-http--downloadother",
-            "WWW:LINK-1.0-http--downloaddata"
+            "IMOS:AGGREGATION--gogoduck",
+            "IMOS:AGGREGATION--bodaac",
+            "IMOS:AGGREGATION--aodaac"
```

Comment:
Yep thats what Nat has reported to me. Should i tell her anything to fix? No more AODACC?